### ISSUE TITLE
Escaping tabbing on keyboard doesn't work on Mac

### ISSUE THREAD

**Issue by microsoftsam** [Microsoft/monaco-editor#465]
Accessibility issue: Escaping tabbing and exiting to main site's tab structure doesn't work on mac. 
On windows: Ctrl + M escapes the tabbing. 

monaco-editor version: 0.8.1
Browser: Chrome
OS: Mac OSX 10.12.1
Steps or JS usage snippet reproducing the issue:

Open https://microsoft.github.io/monaco-editor/playground.html
Set focus on one of the monaco editors 
Type Ctrl + M. 
Continue to tab 

Expected result: tab is escaped after the Ctrl + M, and tabbing continues on the rest of the page. 
Actual result: tab is trapped by the monaco editor, and it continues to add tabs to your file.

Similar to #296 but on Mac.

**Comment by microsoftsam** [comment]
Turns out this issue repro's on VScode on Mac too. I can file the issue there if you like @alexandrudima?

**Comment by microsoftsam** [comment]
Found the solution in: 
https://github.com/Microsoft/vscode/blob/master/src/vs/editor/contrib/toggleTabFocusMode/common/toggleTabFocusMode.ts#L27

Looks like on Mac it's: 
Ctrl + Shift + M
